hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -168,6 +168,8 @@
     @Deprecated
     public static final String PROP_ENABLE_JMX_DETAILED = GroupProperty.ENABLE_JMX_DETAILED.getName();
     @Deprecated
+    public static final String PROP_JMX_UPDATE_INTERVAL_SECONDS = GroupProperty.JMX_UPDATE_INTERVAL_SECONDS.getName();
+    @Deprecated
```

Comment:
Same as in the other PR. `GroupProperty` should be enough, since `GroupProperties` was never intended to be a public API (but we don't know for sure, so I deprecated everything in 3.6).